/navpi_coordinator:
  ros__parameters:
    global_planner_timeout: 5.0
